source/quick-start.txt

```diff
@@ -207,22 +207,35 @@ You can use the ``write()`` method to write objects of the following types to Mo
    write(coll, arrow_table)
    write(coll, ndarrays)
 
-You can add an optional argument to ``write()`` called ``exclude_none``.
-This argument is ``False`` by default. When it is set to ``True``, empty
-values will not be written to the database.
+.. note::
+  
+   NumPy arrays are specified as ``dict[str, ndarray]``.
+
+Ignore Empty Values
+~~~~~~~~~~~~~~~~~~~
+
+The ``write()`` method optionally accepts a boolean ``exclude_none`` parameter.
+If you set this parameter to ``True``, the driver doesn't write empty values to
+the database. If you set this parameter to ``False`` or leave it blank, the 
+driver writes ``None`` for each empty field.
 
-The code in the following example writes an Arrow ``Table`` to MongoDB twice: once
-with ``exclude_none`` defaulting to ``False``, and once with it set to ``True``.
+The code in the following example writes an Arrow ``Table`` to MongoDB twice. One
+of the values in the ``b`` field is set to ``None``. 
+
+The first call to the ``write()`` method omits the ``exclude_none`` parameter, 
+so it defaults to ``False``. All values in the ``Table``, including ``None``, 
+are written to the database. The second call to the ``write()`` method sets 
+``exclude_none`` to ``True``, so the empty value in the ``b`` field is ignored.
```

Comment:
S: Because `b` is a field name and shown as a string in the output, do the same here

```suggestion
The code in the following example writes an Arrow ``Table`` to MongoDB twice. One
of the values in the ``'b'`` field is set to ``None``. 

The first call to the ``write()`` method omits the ``exclude_none`` parameter, 
so it defaults to ``False``. All values in the ``Table``, including ``None``, 
are written to the database. The second call to the ``write()`` method sets 
``exclude_none`` to ``True``, so the empty value in the ``'b'`` field is ignored.
```